<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWidget</class>
 <widget class="QWidget" name="ControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_15">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="5,2">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>输出栏</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QPlainTextEdit" name="outputEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>输入栏</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPlainTextEdit" name="inputEdit"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>欢迎使用本程序(￣▽￣)／</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnClose">
              <property name="text">
               <string>退出(C)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSend">
              <property name="text">
               <string>发送(S)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="3,2,2,2">
       <item>
        <widget class="QGroupBox" name="gpbPort">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>串口配置</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
              <item>
               <widget class="QLabel" name="lblPortName">
                <property name="text">
                 <string>串口：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cmbPortName"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
              <item>
               <widget class="QLabel" name="lblBaudRate">
                <property name="text">
                 <string>波特率：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cmbBaudRate"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
              <item>
               <widget class="QLabel" name="lblDataBits">
                <property name="text">
                 <string>数据位：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cmbDataBits"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
              <item>
               <widget class="QLabel" name="lblParity">
                <property name="text">
                 <string>奇偶校验位：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cmbParity"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
              <item>
               <widget class="QLabel" name="lblStopBits">
                <property name="text">
                 <string>停止位：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cmbStopBits"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
              <item>
               <widget class="QLabel" name="lblFlowControl">
                <property name="text">
                 <string>流控制：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cmbFlowControl"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QPushButton" name="btnOpenPort">
                <property name="text">
                 <string>打开串口</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnClosePort">
                <property name="text">
                 <string>关闭串口</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gpbVolt">
         <property name="title">
          <string>电压表</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLCDNumber" name="lcdNumber"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gpbLED">
         <property name="title">
          <string>LED</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6" stretch="2,1">
            <item>
             <widget class="QLabel" name="lblLED">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>LED：关</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QPushButton" name="btnTestLED">
                <property name="text">
                 <string>测试</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnCloseLED">
                <property name="text">
                 <string>关闭</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gpbBuzz">
         <property name="title">
          <string>蜂鸣器</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8" stretch="2,1">
            <item>
             <widget class="QLabel" name="lblBuzz">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>蜂鸣器：关</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QPushButton" name="btnTestBuzz">
                <property name="text">
                 <string>测试</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnCloseBuzz">
                <property name="text">
                 <string>关闭</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
